### PR TITLE
Setup GitHub Actions.

### PR DESCRIPTION
**Pull request by skmshaffer**
Sets up GitHub Actions for CI on both the Client and Server Projects set to run on both Push and PRs.

This PR is dependent on both #25 and #26 and thus should be reviewed after those two.

### CODE REVIEW

File: src/library-server/.idea/misc.xml [hunk 1]
```diff
@@ -4,8 +4,11 @@
   <component name="FrameworkDetectionExcludesConfiguration">
     <file type="web" url="file://$PROJECT_DIR$" />
   </component>
+<<<<<<< HEAD
+=======
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
+>>>>>>> master
```
Comment (mathvav) [inline src/library-server/.idea/misc.xml:12]: alexa play uhhhh

Comment (skmshaffer) [reply src/library-server/.idea/misc.xml:12]: :(. VS Code said I got them all.

Comment (skmshaffer) [reply src/library-server/.idea/misc.xml]: I have since fixed it.

**Comment by skmshaffer** [comment]
Merging since merge conflict fixed and no other outstanding comments.

**Pull request merged**